assets/app/views/directives/_pod-warnings.html

```diff
@@ -1,6 +1,3 @@
-<span class="pod-warnings">
-  <span class="pficon-layered" title="" data-toggle="popover" data-placement="right" data-trigger="hover" data-html="true">
-    <span class="pficon pficon-warning-triangle"></span>
-    <span class="pficon pficon-warning-exclamation"></span>
-  </span>
+<span class="pod-warnings pficon pficon-warning-triangle-o">
```

Comment:
this one i think the classes got moved up one level too many, `pficon pficon-warning-triangle-o` need to move to the child span.